cfn-resources/project-invitation/cmd/resource/create.go

```diff
@@ -0,0 +1,64 @@
+// Copyright 2023 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package resource
+
+import (
+	"context"
+
+	"github.com/aws-cloudformation/cloudformation-cli-go-plugin/cfn/handler"
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/mongodb/mongodbatlas-cloudformation-resources/profile"
+	"github.com/mongodb/mongodbatlas-cloudformation-resources/util"
+	log "github.com/mongodb/mongodbatlas-cloudformation-resources/util/logger"
+	progressevents "github.com/mongodb/mongodbatlas-cloudformation-resources/util/progressevent"
+	"go.mongodb.org/atlas-sdk/v20230201008/admin"
+)
+
+func CreateOp(req handler.Request, prevModel *Model, currentModel *Model) (handler.ProgressEvent, error) {
+	setup()
+
+	_, _ = log.Debugf("Create() currentModel:%+v", currentModel)
```

Comment:
nit: no need for `_, _` I guess?

Reply:
many other places that do that below

Reply:
_(I know you didn't write this and moved it from other places)_

Reply:
> nit: no need for _, _ I guess?

i think we need it to make clear that the function is returning values but we deliberately ignore them

Reply:
I will update the others as well to make it consistent

Reply:
FYI: I will open a follow-up PR to update the other files with this format if that is okay with you. (there are a lot of files 😅 )

Reply:
I think _, _ is currently needed because of our linter rules

Reply:
I see `log.Printf` that do not have it in the repo 🤔